upgrade/Prepare_for_Upgrade_to_Next_CSM_Major_Version.md

```diff
@@ -50,7 +51,19 @@ CSM version on the system -- not the target version of the upgrade.
 
 See [Check for latest documentation](../update_product_stream/README.md#check-for-latest-documentation) for instructions.
 
-### 3. Export Nexus data
+### 3. Fix Kafka CRD issue
+
+If a cluster which was initially installed with CSM 1.3 or earlier, and is now getting upgraded to CSM 1.6.0 or CSM 1.6.1 from CSM 1.5.x, the upgrade of the `cray-kafka-operator` helm chart would fail as part of the CSM services upgrade process.
+
+**Note:**  This issue would not affect clusters that were freshly installed with CSM 1.4.x or CSM 1.5.x and are being upgraded to CSM 1.6.x versions.
+
+If the cluster has followed the upgrade path mentioned above, run the following script to apply the fix -
+
+[`kafka_crd_fix.sh`](../troubleshooting/scripts/kafka_crd_fix.sh)
```

Comment:
This should give the full path to the script, so they can cut-n-paste it:
```
    ```bash
    /usr/share/doc/csm/troubleshooting/scripts/kafka_crd_fix.sh
    ```
```